jobs/mozillians_magic.rb

```diff
@@ -42,19 +41,22 @@ def query_mozillians(params = {})
       http.request(request)
     end
 
-    def query_mozillians_group(group)
-      response = query_mozillians({ group: group })
-
-      if response.code.to_i == 200
-        res = JSON.parse(response.body)
-        count = res['count']
+    def iterate_mozillians_groups(groups)
+      remove_from_group('mozillians_nda')
```

Comment:
Otherwise, if a user is removed from the nda group in Mozillians, they'll remain in the group in Discourse.